lib/chef/knife/rackspace_server_create.rb

```diff
@@ -411,7 +421,9 @@ def run
           msg_pair('rax_service_level_automation', server.metadata.all['rax_service_level_automation'])
           Chef::Application.fatal! 'Server didn\'t finish on time'
         end
+
         msg_pair("Metadata", server.metadata)
+
         if(networks && Chef::Config[:knife][:rackspace_networks])
           msg_pair("Networks", Chef::Config[:knife][:rackspace_networks].sort.join(', '))
         end
```

Comment:
Perhaps this output should follow the rackconnect setup and actually introspect the `server` object so that it doesn't inadvertently lie to the user.